docker/stringsifter/pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "dockerfiles"
+version = "0.1.0"
+description = ""
+authors = ["TalZich <tzichlinsky@paloaltonetworks.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "~3.10"
+stringsifter = "3.20230711"
+wheel = "*"
+scikit-learn = "1.3.0"
```

Comment:
make sure this is properly ignored in the dependabot config file since we want it to stay locked here

Reply:
done